content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
Yeah, but if this is not used, update this guide and add some explanation for it will be good too.